### PR TITLE
:penguin: Fixes to the orin image

### PR DESCRIPTION
**Pull request by mudler**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Review by mauromorales — DISMISSED** [review]

**Review by Itxaka — DISMISSED** [review]

File: Earthfile [hunk 1]
```diff
@@ -219,7 +219,7 @@ framework:
         COPY overlay/files-fedora/ /framework
     ELSE IF [ "$FLAVOR" = "debian" ] || [ "$FLAVOR" = "ubuntu" ] || [ "$FLAVOR" = "ubuntu-20-lts" ] || [ "$FLAVOR" = "ubuntu-22-lts" ]
         COPY overlay/files-ubuntu/ /framework
-    ELSE IF [[ "$FLAVOR" =~ ^ubuntu-arm* ]]
+    ELSE IF [[ "$FLAVOR" =~ ^ubuntu-arm* ]] || [[ "$FLAVOR" = "ubuntu-20-lts-arm-nvidia-jetson-agx-orin" ]]
```
Comment (mauromorales) [inline Earthfile:222]: I must have made am istake there, it should actually be (not sure about the regex in bash :P )

```suggestion
    ELSE IF [[ "$FLAVOR" =~ ^ubuntu-arm* ]] || [[ "$FLAVOR" = "ubuntu-[0-9]{2}-lts-arm*" ]]
```

**Comment by mudler** [comment]
I don't think regex works at all:

```
    +base-image | --> IF [[ "$FLAVOR" =~ ^alpine.* ]]                                                                                                                                                                                   
         +base-image | /bin/sh: 1: [[: not found 
```

**Review by Itxaka — COMMENTED** [review]
I'm against this just because it removed visibility from the build process. Having it all on the earthly file is much nicer as you can follow up along

**Comment by mudler** [comment]
> I'm against this just because it removed visibility from the build process. Having it all on the earthly file is much nicer as you can follow up along

but regex don't work there? see https://github.com/kairos-io/kairos/pull/1455#issuecomment-1564321686

**Comment by mudler** [comment]
@Itxaka any suggestion? https://github.com/kairos-io/kairos/pull/1455/commits/ae9ed21eac31a9a3946b2cb6059310728d7dec96 doesn't seem to work either

**Comment by mudler** [comment]
> @Itxaka any suggestion? [ae9ed21](https://github.com/kairos-io/kairos/commit/ae9ed21eac31a9a3946b2cb6059310728d7dec96) doesn't seem to work either

ah no.. seems to work, Earthly output is really deceiving

**Comment by mudler** [comment]
```

~/_git/kairos fixups*
base ❯ docker run -ti --rm ttl.sh/kairos-jetson-orin-agx-tests:latest_linux_arm64
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64) and no specific platform was requested
root@98ffa2022410:/# ls
bin  boot  dev  etc  home  lib  media  mnt  opt  proc  root  run  sbin  srv  sys  system  tmp  usr  var
root@98ffa2022410:/# cd /etc/elemental/
root@98ffa2022410:/etc/elemental# ls
config.yaml
root@98ffa2022410:/etc/elemental# cat config.yaml 
cosign: false
verify: false
install:
  grub-entry-name: "kairos"
  system:
    size: 2300
  recovery-system:
    size: 2300
upgrade:
  grub-entry-name: "kairos"
  recovery-system:
    size: 2300
  system:
    size: 2300
reset:
  grub-entry-name: "kairos"
  system:
    size: 2300

cloud-init-paths:
- /run/initramfs/cos-state
```

**Review by Itxaka — APPROVED** [review]

**Pull request merged**